app/main/templates/index.html

```diff
@@ -3,117 +3,86 @@
-            <div class="card-footer">
-              <a href="https://hazen.readthedocs.io/en/latest/tasks.html#spatial-resolution" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'geometric_d.png') }}" class="card-img-top">
-          <div class="card-body">
-            <h5 class="card-title">Geometric Distortion</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://readthedocs.org/projects/hazen/" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
+
+            <div class="hazen-dotted-horizontal-line py-3"></div>
+        </section>
+
+        <!-- Example Images -->
+        <section class="pb-5">
+            <div class="row py-3">
+                <div class="col">
+                    <img src="{{url_for('static', filename = 'sp_res.jpg') }}" class="card-img-top">
+                </div>
+                <div class="col">
+                    <img src="{{url_for('static', filename = 'uniformity.jpg') }}" class="card-img-top">
+                </div>
+                <div class="col">
+                    <img src="{{url_for('static', filename = 'ghost.jpg') }}" class="card-img-top">
+                </div>
             </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'slice_par.jpg') }}" class="card-img-top">
-          <div class="card-body">
-            <h5 class="card-title">Slice Parameters</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://hazen.readthedocs.io/en/latest/tasks.html#slice-position-and-width" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
+
+            <div class="hazen-dotted-horizontal-line py-3"></div>
+        </section>
+
+        <!-- Introductory Text -->
+        <section class="pb-5">
+            <div class="row">
+                <h2 class="h2 pb-3">HAZEN Developers</h2>
+                <p class="p-front-page lead">
+                    Hazen is developed at Guy’s and St Thomas’ NHS Foundation Trust by healthcare scientists in the
+                    Clinical Scientific Computing department in collaboration with the MR Physics team.
```

Comment:
add a link to our GitHub, link users for how to submit issues/complaints and feature requests. Basically how can they get in touch with us.

Reply:
Added new section